src/app.py

```diff
@@ -1,5 +1,15 @@
+from flask import Flask
 import connexion
+from flask_pymongo import pymongo
+import repositories.db as DB
 
 app = connexion.App(__name__, specification_dir='./')
-app.add_api('swagger.yaml')
-app.run(port=8822)
```

Comment:
Also, why is the swagger file removed?

Reply:
Made changes regarding to all your comments, please have a look